### PR TITLE
1.6 merlin

### PR DESCRIPTION
**Pull request by mark-christiaens**

### CODE REVIEW

File: .merlin [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+B _build/src/paxos
+B _build/src/plugins
+B _build/src/system
+B _build/src/tlog
+B _build/src/tools
+
+PKG bz2
+PKG camltc
+PKG lwt
+PKG lwt.ssl
```
Comment (NicolasT) [inline .merlin]: Arakoon 1.6 doesn't depend on `lwt.ssl`, `quickcheck`, `snappy` and `ssl`.

Comment (mark-christiaens) [inline .merlin]: Fixed

**Comment by NicolasT** [comment]
LGTM

**Comment by NicolasT** [comment]
Looks like I failed to spot a missing `B _build/src/node` line. Will fix.

**Pull request merged**